src/providers/sambanova/index.ts

```diff
@@ -0,0 +1,32 @@
+import { chatCompleteParams } from '../open-ai-base';
+import { ProviderConfigs } from '../types';
+import SambaNovaAPIConfig from './api';
+import { SamvaNovaChatCompleteStreamChunkTransform } from './chatComplete';
+
+const SambaNovaConfig: ProviderConfigs = {
+  chatComplete: chatCompleteParams(
+    [
+      'functions',
+      'function_call',
+      'presence_penalty',
+      'frequency_penalty',
+      'logit_bias',
+      'user',
+      'seed',
+      'tools',
+      'tool_choice',
+      'response_format',
+      'logprobs',
```

Comment:
We're expecting the response to send `logprobs` but we're excluding it here, is there any issue here?

> Sorry, I'm missing the documentation with me, so had to confirm.

Reply:
My bad, i will review these as soon as possible. 

if it meant that there are no logprob params, at this moment, there is no mention of these params. i want to believe the docs that i have dont have complete references.

Reply:
My understanding is that the request would have that logprobs arg, but it wasn't documented.

Reply:
Ok, for now we can remove the `logprobs` i.e don't expect it to be available in the response and don't accept it in the request body as well. Once they are with good documentation/ available for everyone we can make the change again.